example-code/cypress/bookstore/bookstore.js

```diff
@@ -0,0 +1,18 @@
+export const bookstore ={
+    LB_USERNAME: "#userName-value",
+    BTN_OK: "#closeSmallModal-ok",
+    TB_BOOK: '.ReactTable a',
+
+    addBook(book){
+        cy.get(this.TB_BOOK).contains(book).click();
+        cy.get('button').contains('Add to your collection').click();

```

Comment:
`Add to your collection` button is belong to book detail page. And all the pages file should be placed inside `pages` folder